client/login/magic-login/index.jsx

```diff
@@ -99,6 +107,7 @@ class MagicLogin extends React.Component {
 			isJetpack: isJetpackLogin,
 			isGutenboarding: isGutenboardingLogin,
 			locale: locale,
+			...( signupUrl && { signupUrl } ),
```

Comment:
This can be simply `signupUrl`. The `login()` function can handle a null-ish `signupUrl` parameter. It doesn't need to be completely omitted.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/54646/commits/032fd7d1c84cd74010825d8fa72b504c0b7d98c4